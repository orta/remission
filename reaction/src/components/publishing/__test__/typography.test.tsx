import * as React from "react"
import * as renderer from "react-test-renderer"

import "jest-styled-components"

import Typography from "../__stories__/typography_examples"

it("renders properly", () => {
  const typography = renderer.create(<Typography />).toJSON()
  expect(typography).toMatchSnapshot()
})
