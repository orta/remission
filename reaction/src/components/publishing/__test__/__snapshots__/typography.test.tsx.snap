// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
.c9 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 15px;
  line-height: 1.25em;
  margin-bottom: 20px;
}

.c8 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 17px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c7 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 19px;
  line-height: 1.5em;
  margin-bottom: 20px;
}

.c6 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 23px;
  line-height: 1.5em;
  margin-bottom: 20px;
}

.c5 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 28px;
  line-height: 1.2em;
  margin-bottom: 20px;
}

.c4 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 30px;
  line-height: 1.25em;
  margin-bottom: 20px;
}

.c3 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 34px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c2 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 37px;
  line-height: 1.2em;
  margin-bottom: 20px;
}

.c1 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 40px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c0 {
  font-family: 'Adobe Garamond W08',  'adobe-garamond-pro',  'AGaramondPro-Regular',  'Times New Roman',  'Times',  'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 50px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c11 {
  font-family: 'ITC Avant Garde Gothic W04',  'AvantGardeGothicITCW01D 731075',  'AvantGardeGothicITCW01Dm',  'Helvetica',  'sans-serif';
  -webkit-font-smoothing: antialiased;
  -webkit-text-transform: uppercase;
  text-transform: uppercase;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 11px;
  line-height: 1.65em;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  margin-bottom: 20px;
}

.c10 {
  font-family: 'ITC Avant Garde Gothic W04',  'AvantGardeGothicITCW01D 731075',  'AvantGardeGothicITCW01Dm',  'Helvetica',  'sans-serif';
  -webkit-font-smoothing: antialiased;
  -webkit-text-transform: uppercase;
  text-transform: uppercase;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  font-size: 13px;
  line-height: 1.65em;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  margin-bottom: 20px;
}

.c23 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 12px;
  line-height: 1.4em;
  margin-bottom: 20px;
}

.c22 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 14px;
  line-height: 1.4em;
  margin-bottom: 20px;
}

.c21 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 16px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c20 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 19px;
  line-height: 1.5em;
  margin-bottom: 20px;
}

.c19 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 32px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c18 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 34px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c17 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 40px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c16 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 45px;
  line-height: 1.2em;
  margin-bottom: 20px;
}

.c15 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 65px;
  line-height: 1em;
  margin-bottom: 20px;
}

.c14 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 80px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c13 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 100px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

.c12 {
  font-family: Unica77LLWebRegular ,      'Arial',      'serif';
  -webkit-font-smoothing: antialiased;
  font-size: 130px;
  line-height: 1.1em;
  margin-bottom: 20px;
}

<div>
  <div
    className="c0"
  >
    Garamond s50: 50px / 1.1em
  </div>
  <div
    className="c1"
  >
    Garamond s40: 40px / 1.1em
  </div>
  <div
    className="c2"
  >
    Garamond s37: 37px / 1.2em
  </div>
  <div
    className="c3"
  >
    Garamond s34: 34px / 1.5em
  </div>
  <div
    className="c4"
  >
    Garamond s30: 30px / 1.25em
  </div>
  <div
    className="c5"
  >
    Garamond s28: 28px / 1.5em
  </div>
  <div
    className="c6"
  >
    Garamond s23: 23px / 1.5em
  </div>
  <div
    className="c7"
  >
    Garamond s19: 19px / 1.1em
  </div>
  <div
    className="c8"
  >
    Garamond s17: 17px / 1.1em
  </div>
  <div
    className="c9"
  >
    Garamond s15: 15px / 1.25em
  </div>
  <hr />
  <br />
  <div
    className="c10"
  >
    Avant Garde s13: 13px / 1.65em
  </div>
  <div
    className="c11"
  >
    Avant Garde s11: 11px / 1.65em
  </div>
  <hr />
  <div
    className="c12"
  >
    Unica s130: 130px / 1.1em
  </div>
  <div
    className="c13"
  >
    Unica s100: 100px / 1.1em
  </div>
  <div
    className="c14"
  >
    Unica s80: 80px / 1.1em
  </div>
  <div
    className="c15"
  >
    Unica s65: 65px / 1em
  </div>
  <div
    className="c16"
  >
    Unica s45: 45px / 1.1em
  </div>
  <div
    className="c17"
  >
    Unica s40: 40px / 1.1em
  </div>
  <div
    className="c18"
  >
    Unica s34: 34px / 1.1em
  </div>
  <div
    className="c19"
  >
    Unica s32: 32px / 1.1em
  </div>
  <div
    className="c20"
  >
    Unica s19: 19px / 1.5em
  </div>
  <div
    className="c21"
  >
    Unica s16: 16px / 1.1em
  </div>
  <div
    className="c22"
  >
    Unica s14: 14px / 1.1em
  </div>
  <div
    className="c23"
  >
    Unica s12: 12px / 1.1em
  </div>
</div>
`;
